nilearn/glm/tests/test_thresholding.py

```diff
@@ -16,146 +16,159 @@
+            height_control=control,
+            cluster_threshold=5,
+        )
         vals = get_data(th_map)
         assert np.sum(vals > 0) == 8
 
     # test 5: direct threshold
     th_map, _ = threshold_stats_img(
-        stat_img, mask_img, threshold=4.0, height_control=None,
-        cluster_threshold=0)
+        stat_img,
+        mask_img,
+        threshold=4.0,
+        height_control=None,
+        cluster_threshold=0,
+    )
     vals = get_data(th_map)
     assert np.sum(vals > 0) == 8
 
     # test 6: without mask
     th_map, _ = threshold_stats_img(
-        stat_img, None, threshold=4.0, height_control=None,
-        cluster_threshold=0)
+        stat_img, None, threshold=4.0, height_control=None, cluster_threshold=0
+    )
     vals = get_data(th_map)
     assert np.sum(vals > 0) == 8
 
     # test 7 without a map
     th_map, threshold = threshold_stats_img(
-        None, None, threshold=3.0, height_control=None,
-        cluster_threshold=0)
+        None, None, threshold=3.0, height_control=None, cluster_threshold=0
+    )
     assert threshold == 3.0
     assert th_map is None
 
     th_map, threshold = threshold_stats_img(
-        None, None, alpha=0.05, height_control='fpr',
-        cluster_threshold=0)
-    assert (threshold > 1.64)
+        None, None, alpha=0.05, height_control="fpr", cluster_threshold=0
+    )
+    assert threshold > 1.64
     assert th_map is None
 
     with pytest.raises(ValueError):
-        threshold_stats_img(None, None, alpha=0.05, height_control='fdr')
+        threshold_stats_img(None, None, alpha=0.05, height_control="fdr")
     with pytest.raises(ValueError):
-        threshold_stats_img(None, None, alpha=0.05,
-                            height_control='bonferroni')
+        threshold_stats_img(
+            None, None, alpha=0.05, height_control="bonferroni"
+        )
 
     # test 8 wrong procedure
     with pytest.raises(ValueError):
-        threshold_stats_img(None, None, alpha=0.05, height_control='plop')
+        threshold_stats_img(None, None, alpha=0.05, height_control="plop")
 
 
 def test_hommel():
     # Check that the computation of Hommel value
     # For these, we take the example in  Meijer et al. 2017
     # 'A shortcut for Hommel's procedure in linearithmic time'
     # and check that we obtain the same values
-    z = norm.isf([1.e-8, .01, .08, .1, .5, .7, .9])
-    assert _compute_hommel_value(z, 1.e-9) == 7
-    assert _compute_hommel_value(z, 1.e-7) == 6
-    assert _compute_hommel_value(z, .059) == 6
-    assert _compute_hommel_value(z, .061) == 5
-    assert _compute_hommel_value(z, .249) == 5
-    assert _compute_hommel_value(z, .251) == 4
-    assert _compute_hommel_value(z, .399) == 4
-    assert _compute_hommel_value(z, .401) == 3
-    assert _compute_hommel_value(z, .899) == 3
-    assert _compute_hommel_value(z, .901) == 0
+    z = norm.isf([1.0e-8, 0.01, 0.08, 0.1, 0.5, 0.7, 0.9])
+    assert _compute_hommel_value(z, 1.0e-9) == 7
```

Comment:
Screams "test parametrization" so loud ! :laughing: (To be tackled in another PR I suppose...)